src/You%27re_Offline.html

```diff
@@ -0,0 +1,44 @@
+<!---{
+  "hideCode": true,
+  "hidePreview": true
+}--->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+
+  <style amp-custom>
+  #dinosaur {
+    margin-top: 10vh;
+    margin-bottom: 16px;
+  }
+  .box p {
+    margin-top: 16px;
+  }
+@media (max-width: 1024px) {
+  #dinosaur {
+    margin-top: 0;
+  }
+}
+  </style>
+
+</head>
+<body>
+<!--
+#### Serviceworker to the rescue
```

Comment:
I think "Serviceworker" is better styled as "ServiceWorker" or "Service Worker" or "service worker"